Models/CandidateTestModel.ts

```diff
@@ -0,0 +1,20 @@
+export const DefaultExecutionTimeLimit = "4 seconds";
+export const DefaultJsTips = "// Prints help message to the console\n" + "Returns a string\n" + "function helloWorld(name) {\n" + "    console.log(\"This prints to the console when you Run Tests\");\n" + "    return \"Hello, \" + name;\n" + "}";
+
+
+export interface CandidateTestModel {
+    title: string;
+    brief: string;
+    example: string;
+    number: number;
+    modelAnswer: string;
+    inputType: string;
+    inputDescription: string;
+    input: string | number | string[] | number[];
```

Comment:
this still looks wrong - we don't handle arrays in the JSX

Reply:
I'll remove for now